### PR TITLE
Wire up delete to the explore page

### PR DESCRIPTION
**Pull request by white0ut**
Will add a dev mode that spins up local DB and initializes it with data next. Found myself doing that a lot.

### CODE REVIEW

**Comment by white0ut** [comment]
@dudasaus what is even going on

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ View it live at [dudasaus.github.io/tierlist/](https://dudasaus.github.io/tierli
 git config --local core.hooksPath .githooks/
 ```
 
-## Front end
+## Frontend
```
Comment (dudasaus) [inline README.md:11]: I can't endorse this change, but i'll allow it

File: frontend/src/routes/explore/ExploreRoute.tsx [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
         name: t.name ?? 'undefined',
+        id: t.id ?? 'undefined',
       }));
     },
   });
 
+  const deleteTemplate = useMutation({
+    mutationFn: (id: string) => ClientService.deleteTemplate({ id: id }),
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: ['list-templates'] });
```
Comment (dudasaus) [inline frontend/src/routes/explore/ExploreRoute.tsx:25]: I'm not 100% up to speed on how all this works, but i think we can also invalidate [this query key](https://github.com/dudasaus/tierlist/blob/main/frontend/src/routes/tier_list/TierListRoute.tsx#L13). IDK the syntax off the top of my head yet.

**Review by dudasaus — APPROVED** [review]
Changes are all nits - I can do a pass over and send your way to show what I would have done differently after, no worries either way

File: README.md (hunk 1 above)
Comment (white0ut) [reply README.md:11]: :gasp: I thought it was an accounting error.. I'm disappointed Austin..

File: frontend/src/routes/explore/ExploreRoute.tsx (hunk 2 above)
Comment (white0ut) [reply frontend/src/routes/explore/ExploreRoute.tsx:25]: I tried

```
queryClient.invalidateQueries({ queryKey: ['list-templates', 'template'] });
```

and it caused a big delay from when I hit the delete the button and it actually happening. If I have another window with the template open that I deleted, it picks up on it in ~2 seconds.

**Pull request merged**